dll/win32/shdocvw/mrulist.cpp

```diff
@@ -429,51 +575,144 @@ class CMruLongList
 
 HRESULT CMruLongList::_InitSlots()
 {
-    FIXME("Stub\n");
-    return E_NOTIMPL;
+    DWORD cbData = (m_cSlotRooms + 1) * sizeof(UINT);
+    m_puSlotData = (UINT*)LocalAlloc(LPTR, cbData);
+    if (!m_puSlotData)
+        return E_OUTOFMEMORY;
+
+    if (SHGetValueW(m_hKey, NULL, L"MRUListEx", NULL, m_puSlotData, &cbData) == ERROR_SUCCESS)
+        m_cSlots = (cbData / sizeof(UINT)) - 1;
+    else
+        _ImportShortList();
+
+    m_puSlotData[m_cSlots] = 0xFFFFFFFF;
```

Comment:
"0xFFFFFFF" ? Not `NULL` or `(UINT*)(-1)` (since that value seems to be something like "reserved but unallocated pointer" of some sort.... :thinking: )

Reply:
This is not a pointer but a `DWORD` value.

Reply:
ok, then maybe use `(-1)` or `ULONG_MAX` instead of the hexadecimal form, here.

Reply:
Used `MAXDWORD`.